File: simpsons/lib/models/character.dart
class Character {
  final String name;
  final String imageUrl;
  final String description;

  Character({
    required this.name,
    required this.imageUrl,
    required this.description,
  });
  
  factory Character.fromJson(Map<String, dynamic> json) {
    return Character(
      name: json['name'] as String? ?? 'Unknown',
      imageUrl: json['image_url'] as String? ?? '',
      description: json['description'] as String? ?? '',
    );
  }
}

File: simpsons/backend/src/models/Character.ts
import { Schema, model } from 'mongoose';

interface Character {
    name: string;
    description: string;
    image_url: string;
}

const CharacterSchema = new Schema<Character>({
    name: { type: String, required: true },
    description: { type: String, required: true },
    image_url: { type: String, required: true }
});

export default model<Character>('Character', CharacterSchema);